CMakeLists.txt

```diff
@@ -279,7 +279,7 @@ include(CheckLibraryExists)
 find_package(LibUV REQUIRED)
 include_directories(SYSTEM ${LIBUV_INCLUDE_DIRS})
 
-find_package(Msgpack REQUIRED)
+find_package(Msgpack 1.0.0 REQUIRED)
```

Comment:
It's still useful to avoid setting the `PC_*` variables to incorrect paths, since those get used as hints.